src/model/RacingGame.js

```diff
@@ -0,0 +1,62 @@
+const RacingCar = require('./RacingCar');
+
+class RacingGame {
+  #cars;
+  #winners;
+
+  constructor(names) {
+    this.#cars = [];
+    for (const name of names) {
+      this.#cars.push(new RacingCar(name));
+    }
+    this.#winners = [];
+  }
```

Comment:
```suggestion
  constructor(names) {
    this.#cars = names.map(name => new RacingCar(name))
    this.#winners = [];
  }
```
배열 만들 때는 가능한 map 쓰는게 좋아. 가독성 측면에서도 그렇고 확실하진 않지만 성능에서도 그렇고.
map이랑 for문, for...of, for...in 등 성능 비교한거 한번 찾아봐